spacy/cli/project/dvc.py

```diff
@@ -105,13 +105,15 @@ def update_dvc_config(
         dvc_config_path.unlink()
     dvc_commands = []
     config_commands = {cmd["name"]: cmd for cmd in config.get("commands", [])}
-    for name in workflows[workflow]:
+    processed_step = False
+    for name in get_workflow_steps(workflows[workflow]):
```

Comment:
As far as I could find out DVC does not offer anything comparable to multiprocessing groups, so that they need to be 'flattened out' when converting a spaCy config file to a DVC config file.

Reply:
We could also consider raising an error rather than flattening in this case?